mysite/notes/models.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+from django.utils import timezone
+import datetime
+
+
+class MyNotes(models.Model):
+    title = models.CharField(max_length=60)
```

Comment:
explicitly mark it required for both title and description.

Reply:
Alright.